modules/promotions/admin-menu-items/custom-code-promotion-item.php

```diff
@@ -20,25 +20,34 @@ public function get_page_title() {
 	}
 
 	public function get_promotion_title() {
-		return esc_html__( 'Add Your Custom Code', 'elementor' );
+		echo sprintf(
+			esc_html( 'Enjoy Creative Freedom with Custom Code', 'elementor' ),
+			'<br />'
+		);
 	}
 
-	public function get_promotion_description() {
-		return esc_html__(
-			'Custom Code is a tool gives you one place where you can insert scripts, rather than dealing with dozens of different plugins and deal with code.',
-			'elementor'
-		);
+	public function set_list() {
+		return [
+			esc_html__( 'Add Custom Code snippets anywhere on your website, including the header or footer to measure your page’s performance*', 'elementor' ),
+			esc_html__( 'Use Custom Code to create sophisticated custom interactions to engage visitors', 'elementor' ),
+			esc_html__( 'Leverage Elementor AI to instantly generate Custom Code for Elementor', 'elementor' ),
+		];
 	}
```

Comment:
Tweak in [ED-13558]. David and I brought get_description() back in the base to make the text area more flexible for future changes. 

```suggestion
public function get_description(): string {
		return $this->create_list( [
			esc_html__( 'Add Custom Code snippets anywhere on your website, including the header or footer to measure your page’s performance*', 'elementor' ),
			esc_html__( 'Use Custom Code to create sophisticated custom interactions to engage visitors', 'elementor' ),
			esc_html__( 'Leverage Elementor AI to instantly generate Custom Code for Elementor', 'elementor' ),
		] );
	}
```

Reply:
@Barcov - i saw this in pro, are you sure we have the same in core?

Reply:
@TzviRabinovitch We moved the core base to pro because of BC concerns. Louis, David and I discussed BC concerns and the content of the base and that we'll make changes and David would have rolled these into core as well. Ideally these two bases should be the same since we're doing exactly the same thing in core and pro. Right now pro is the most up to date on what we landed on as a team.

Dropping get_description() makes a lot of rework if there's a change request for anything else but a <li> in these promotions and it also makes the filter for this area less flexible. Changing the signature of the base was also a reason why we ended up with two bases. David and I decided on this format instead to allow html returns from the filter in the description.